i18n/zh-Hant.json

```
@@ -0,0 +1,32 @@
+{
+  "bookmarks": "書籤",
+  "composeNewTweet": "推文",
+  "explore": "搜尋和探索",
+  "followers": "追隨者",
+  "following": "追隨",
+  "lists": "列表",
+  "profile": "個人資料",
+  "showNewMulti": "顯示 $ 新推文",
+  "showNewSingle": "顯示 1 個新推文",
+  "translateTweet": "翻譯推文",
+  "translatedTweetInfo": "由 $source$ 從 $lang$ 翻譯",
+  "tweets": "推文",
```

Some of the default strings are not needed anymore, I found a way to include them natively
```suggestion
  "showNewMulti": "顯示 $ 新推文",
  "showNewSingle": "顯示 1 個新推文",
  "translatedTweetInfo": "由 $source$ 從 $lang$ 翻譯",
  
```